iis-wf/iis-wf-referenceextraction/src/main/resources/eu/dnetlib/iis/wf/referenceextraction/project/tara/oozie_app/workflow.xml

```diff
@@ -32,7 +32,7 @@
         </property>
         <property>
             <name>number_of_emitted_files</name>
-            <value>100</value>
+            <value>1000</value>
```

Comment:
As we have discussed during today's call: let's get rid of this number and rely on `spark.sql.shuffle.partitions` as the number of output files (what should happen automatically when dropping coalesce).